services/wordpress/wordpress-platform.tester.js

```diff
@@ -0,0 +1,60 @@
+'use strict'
+
+const Joi = require('joi')
+const ServiceTester = require('../service-tester')
+
+const { isVPlusDottedVersionAtLeastOne } = require('../test-validators')
+
+const t = new ServiceTester({
+  id: 'wordpress',
+  title: 'Wordpress Platform Tests',
+})
+module.exports = t
+
+t.create('Plugin Required WP Version')
+  .get('/plugin/wp-version/akismet.json')
+  .expectJSONTypes(
+    Joi.object().keys({
+      name: 'wordpress',
+      value: isVPlusDottedVersionAtLeastOne,
+    })
+  )
+
+t.create('Plugin Tested WP Version')
+  .get('/plugin/tested/akismet.json')
+  .expectJSONTypes(
+    Joi.object().keys({
+      name: 'wordpress',
+      value: Joi.string().regex(/^v\d+(\.\d+)?(\.\d+)? tested$/),
+    })
+  )
```

Comment:
One final thing:

If we're using standard helpers like `floorCount()`, `versionColor()` etc to generate the colours, we don't generally test the colours in the service tests. Those functions have their own test suite.

When we've got custom colour logic in badges, its useful to nail that down by writing a few tests that use a mock response to return a known value. I know we didn't have any in place before you started (I think the test suite for WordPress was a bit on the light side :disappointed:  ), but would you be able to add some additional tests for the `WordpressPluginTestedVersion` badge. There's some docs on how to mock responses and test colours in the testing docs: https://github.com/badges/shields/blob/master/doc/service-tests.md#5-mocking-responses and plenty of examples you can reference in other service tests.

Reply:
I keep on getting an inaccessible error, I believe it to be because the network gets turned off when I have to do two requests.

I also have school and I can't work on this too much so development will be slow.